pkg/clustermanager/provider.go

```diff
@@ -73,7 +73,7 @@ func CreateProvider(c *controller, providerName string, providerType lcv1alpha1a
 	c.lock.Lock()
 	defer c.lock.Unlock()
 
-	discoveryPrefix := "*"
+	discoveryPrefix := ""
```

Comment:
If the later code
```
			if discoveryPrefix == "" {
				discoveryPrefix = "*"
			}
```
gets deleted then you could also delete the later code in `func (p *provider) filterOut`
```
	if p.discoveryPrefix == "*" {
		return false
	}
```
right?  Does this value get used anywhere else?

Reply:
Correct, the two pieces of code are co-dependent.  I don't think '*' is used elsewhere.